v2/slogger/rolling_file_appender/error.go

```diff
@@ -0,0 +1,157 @@
+package rolling_file_appender
+
+import (
+	"fmt"
+)
+
+type CloseError struct {
```

Comment:
Is this being used at all? Closing a `RollingFileAppender` seems to just surface the `os.File.Close` error if it gets to that point.